src/ctim/schemas/vulnerability.cljc

```diff
@@ -327,17 +357,21 @@
                                "in the CPE.")))))
 
 (def-map-type CPELeafNode
-  (f/required-entries
-   (f/entry :operator cpe-node-operator-string)
-   (f/entry :cpe_match (f/seq-of CPEMatch))))
+  (concat
+   (f/required-entries
+    (f/entry :operator cpe-node-operator-string)
+    (f/entry :cpe_match (f/seq-of CPEMatch)))
+   (f/optional-entries
+    (f/entry :negate f/any-bool))))
```

Comment:
I forgot to ask for it in previous PR about CPE. Please, could you add the description entries and update[ the documentation](https://github.com/threatgrid/ctim/blob/master/doc/structures/vulnerability.md) with `lein doc`?